### ISSUE TITLE
How do you add an external harddrive to the containers backup

### ISSUE THREAD

**Issue by paradoxgrowth** [jlesage/docker-crashplan-pro#326]
Hello Everyone,

I am using a Crashplan container to back up my files and I would like to include an external hard drive to my backup. Unfortuanly I can't seem to figure out how to add it to the shared folders list. I tried the following command:
`-v` /share/external/DEV3301:ro `\`
But I get the error below. What am I doing wrong?
docker: Error response from daemon: invalid volume specification: '/share/external/DEV3301:ro'.

**Comment by jlesage** [comment]
You need to specify the container path in the parameter.  For example:
```
-v /share/external/DEV3301:/DEV3301:ro
```
maps `/share/external/DEV3301` from the host to `/DEV3301` inside the container.

**Comment by paradoxgrowth** [comment]
That did the trick, thanks so much for your help!! By the way, given you can remap store to the container in any way you like will come in handy with CPs new deletion policy. To make an archive simply remap! It will be marked as missing and never be deleted. 

Joel Persinger, PhD
Human Factors Expert
(Sent from my iPhone)

**Comment by madfusker** [comment]
Just be aware that I have seen that external device path change occasionally, especially after some firmware updates.

**Comment by paradoxgrowth** [comment]
Oh, that may not be good, if it re-maps to an old location (i.e. a missing drive), it will mark missing items for deletion in 90 days. Is it possible that might happen?


Joel Persinger

**Comment by madfusker** [comment]
Yes, that is possible.  But what you will see is the mapped location will be empty.  It's always good to watch your crashplan GUI page and make sure all Backup Sets complete successfully.

Bottom paragraph:
[https://helpdesk.code42.com/hc/en-us/articles/4408123149463-Deleted-files-FAQ?mkt_tok=NzYwLU9NVS00NzgAAAF_kz_JxuqdT6yu2Y17AXFbnzJp4-OINK9tpCGOqA1PG6vH_DFxy9FpGK7wvYDBUIoPI4HTksQtbqbhuhA3fVg2eqn3_shko3ynnV6nHBsAOuGEZw](url)

**Comment by paradoxgrowth** [comment]
Hi is it possible to mount an internal folder as an external drive? Is it also possible to specify the mount point? that way data is not overwritten. Ie. to avoid data to be marked as deleted rather than missing. 

I don’t know if I am describing this well so let me communicate the end goal. I like to toss things up to Crashplan and clear it from my hdd for long term storage. We can no longer do that due to crash plans deletion policy change (deletions are no longer kept forever, only 90 days). A loop hole is the “missing” folders (hdd is removed, external is not connected, etc) policy that nothing will ever get deleted if it is flagged missing.
